sdks/python/examples/bulk_fanout/bulk_trigger.py

```diff
@@ -1,33 +1,30 @@
 import asyncio
 
-from hatchet_sdk import new_client
-from hatchet_sdk.clients.admin import TriggerWorkflowOptions, WorkflowRunDict
+from examples.bulk_fanout.worker import ParentInput, bulk_parent_wf
+from hatchet_sdk import Hatchet
+from hatchet_sdk.clients.admin import TriggerWorkflowOptions
 
+hatchet = Hatchet()
 
-async def main() -> None:
 
-    hatchet = new_client()
-
-    workflow_runs = [
-        WorkflowRunDict(
-            workflow_name="BulkParent",
-            input={"n": i},
-            options=TriggerWorkflowOptions(
-                additional_metadata={
-                    "bulk-trigger": i,
-                    "hello-{i}": "earth-{i}",
-                }
-            ),
-        )
-        for i in range(20)
-    ]
-
-    workflowRunRefs = hatchet.admin.run_workflows(
-        workflow_runs,
+async def main() -> None:
+    workflow_run_refs = bulk_parent_wf.run_many(
```

Comment:
this is a bit confusing to me

Reply:
the `run_many`? is it the naming? I wasn't sure what to call this 😅 open to ideas here